### DISCUSSION TITLE
Entity Hider - hide players but still make them visible

### DISCUSSION THREAD

**Discussion by acdoussan** [runelite/runelite#16689] [Ideas]
Often when doing a farm run I'll try to put a seed in the patch, but there is another player under me, and i use the seed on them instead of the patch.

This can be fixed by using entity hider to hide all of the other users, but this is an online game and I like seeing other players. I don't want to run around on "empty" worlds.

It would be nice if entity hider had an option to "remove" other players "technically" so that I can not interact with them and the game doesn't think they are there, but still have their player models shown so that this doesn't look like a dead game.

**Comment by YvesW** [comment]
Give ``Custom Menu Swaps`` by geheur a try. Not entirely sure if it'll work since players swaps are disallowed, but you can likely force prio the patch.
``use,* -> Allotment``
``use,* -> Herb patch``
``use,* -> Flower patch``
``use,* -> *compost bin``
